uploadNeuroDB/register_processed_data.pl

```diff
@@ -354,9 +447,21 @@ sub getAcqProtID    {
     return  ($acqProtID);
 }
 
+
 =pod
-This function parses the mincheader and look for specific field's value.
+
+=head3 fetchMincHeader($file, $field)
+
+This function parses the MINC header and looks for specific field's value.
+
```

Comment:
..and looks for a specific field value

Reply:
@nicolasbrossard Thanks! Sorry I missed it. It has been changed.